domain-cc/README.md

```diff
@@ -58,6 +58,26 @@ Run the tests
 pytest
 ```
 
+## Database Migrations
+This application uses alembic to perform any database migrations or updates.  To perform migrations follow the below steps:
+```
+# Run vro application from abd-vro directory
+./gradlew :dockerComposeUp
+
+# Navigate to cc-app directory
+cd domain-cc/cc-app
+
+# Apply database migrations
+alembic upgrade head
+```
+
+If there are any updates needed for the database (new tables, remove tables, add fields, etc.) you can use alembic to generate the revision file and apply the migrations following the below steps.
+
+1. Update the data models in `abd-vro/domain-cc/cc-app/alembic/db_models.py`
+2. From `cc-app` directory run `alembic revision --autogenerate -m <add note about changes>`
+3. Run `alembic upgrade head` to apply migrations
```

Comment:
I was able to update my database and create a new migration based on these instructions e.g. [adding va.gov claim id](https://github.com/department-of-veterans-affairs/abd-vro/commit/315d03fdc797f1ad2447406b7dc2ca7a87cd1f28)

These were created in the `public` schema for my local machine